src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -275,7 +281,8 @@ private RouteFW resolveTarget(
         private DecoderState decodeState;
         private int slotIndex = NO_SLOT;
         private int slotLimit;
-        private HashMap<String, MqttServerStream> mqttStreams;
+        private HashMap<String, MqttServerStream> mqttSubscriptionStreams;
+        private HashMap<String, MqttServerStream> mqttPublishStreams;
```

Comment:
Please change these types to `private final Map<String, MqttServerStream>` instead.
Suggest renaming the fields to `subscribers` and `publishers`.